capabilities.go

```diff
@@ -34,3 +35,14 @@ func (caps *capabilities) isBlockDeviceSupported() bool {
 func (caps *capabilities) setBlockDeviceSupport() {
 	caps.flags = caps.flags | blockDeviceSupport
 }
+
+func (caps *capabilities) isBlockDeviceHotplugSupported() bool {
```

Comment:
This (and the `qemu.go` changes as a minimum) need corresponding unit tests.

Reply:
I have tests for capabilities and qemu capabilities. I am planning to add tests for the actual qmp commands by starting a an actual qemu instance. I'll add those tests in a separate PR.